### PR TITLE
Reflect merge commit message through to `-content` repo

### PR DESCRIPTION
**Pull request by jebw**
### Trello card

N/a

### Context

Currently we just note the changed sha in the commits we create into the `-content` repo. If these are to be included in release notes than it would help to include further information about what was changed in the `-app` repo.

### Changes proposed in this pull request

Copy the latest commit message in `-app` to the commit being created in `-content`.

### Guidance to review

This is on a branch at the moment - so I've disabled the push to `-content` and the check for `-master`.

Once reviewed I'll re-enable both of those, it'll require re-approval because its changed but than it can be merged

### CODE REVIEW

**Review by sfawcett123 — COMMENTED** [review]
do you needs to get the commit ?
    echo ::set-output name=message::$(git show -s --format=%B HEAD)
there should be a context..
     ${{ github.event.commits[0].message }}

**Comment by jebw** [comment]
Ah thats better - tried dumping the github  object and it didn't show me anything - I'll update

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -128,9 +128,12 @@ jobs:
   update_content_repo:
     name: Update the base image used in the Content repo
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
+    # if: github.ref == 'refs/heads/master'
```
Comment (sfawcett123) [inline .github/workflows/build.yml:131]: I am assuming this is for Debug

**Review by sfawcett123 — DISMISSED** [review]
LGTM

File: .github/workflows/build.yml (hunk 1 above)
Comment (jebw) [reply .github/workflows/build.yml:131]: YUp

**Review by sfawcett123 — APPROVED** [review]

**Pull request merged**